app/sync/goTo.js

```diff
@@ -35,10 +35,17 @@ exports.create = function (api) {
       const locationId = api.app.sync.locationId(loc)
 
       var page = tabs.get(locationId)
+
       if (page) {
         tabs.select(locationId)
         api.history.sync.push(loc)
 
+        if (loc.action == "quote" && page.firstChild && page.firstChild.addQuote) {
+          page.firstChild.addQuote(loc.value)
+          tabs.currentPage().scroll('last')
+        } else if (loc.action == "reply")
+          tabs.currentPage().scroll('last')
```

Comment:
if the thread is not already open the scroll will hit a race condition (I think) where the thread won't all be loaded, and so this action will cause a scroll to the last child currently rendered.... maybe

I guess the question is what is the context in which a person hits this quote or reply button.

There's also a thing I added you should know about where when a person goes to a message location, if that location is a message and it has a root, I tell the thread to render the root, and store the location as the root (because this means any other clicks with the same root will open that tab). Not sure it's relevant here but it's a funny little trick